go/vt/vtgate/schema/update_controller_flaky_test.go

```diff
@@ -177,3 +177,153 @@ func TestMultipleUpdatesFromDifferentShards(t *testing.T) {
 		})
 	}
 }
+
+func TestViewsUpdates(t *testing.T) {
```

Comment:
Do we know why these are flaky tests? I want to tackle the ones we've marked as flaky some day...

Reply:
I believe the reason it is flaky is sometimes health check updates from the shard can be consolidated into a single call if some go-routine is delayed to run the fetch call.